agent/acs/handler/acs_handler.go

```diff
@@ -194,6 +194,13 @@ func (acsSession *session) Start() error {
 			seelog.Debugf("Received connect to ACS message")
 			// Start a session with ACS
 			acsError := acsSession.startSessionOnce()
+			select {
+			case <-acsSession.ctx.Done():
+				seelog.Debug("ACS session context cancelled")
```

Comment:
nit : Should this be same log as below 'ACS session exited cleanly.' and the one other one in this file